### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by mengjingyouling** [thuml/Transfer-Learning-Library#169]
HI，

Can the learning rate of discriminator be updated? Because in the function class DomainDiscriminator(nn.Sequential): **lr=1.**Does the learning rate(1.0) can update follow the epoch？

def get_parameters(self) -> List[Dict]:
    return [{"params": self.parameters(), **"lr": 1.**}]

Such as we adopt the following learning rate updating method：
lr_scheduler_ad = LambdaLR(
    ad_optimizer, lambda x: args.lr * (1. + args.lr_gamma * float(x)) ** (-args.lr_decay))

Because we found in the experiment that the discrimination loss remained stable when the epoch was very small, although the task loss was still decreasing.

Thank you

**Comment by thucbx99** [comment]
By default, the learning rate of discriminator is updated with training. For example, in [dann.py](https://github.com/thuml/Transfer-Learning-Library/blob/master/examples/domain_adaptation/image_classification/dann.py), line `78-81`. The value `1` in function `get_parameters` ([domain_discriminator.py](https://github.com/thuml/Transfer-Learning-Library/blob/4df94ed9d13b6777506118d84c4aecfd1ef878b6/tllib/modules/domain_discriminator.py#L11)) is similar to the initial learning rate and works together with `LambdaLR` to implement specific learning rate strategy.
